index.html

```diff
@@ -843,6 +861,100 @@ <h4>Profile Designator</h4>
           </table>
         </section>
       </section>
+      <section id="time-constraints-section">
+        <h3>Timing constraints</h3>
+        <p>Within a DAPT Script, the following constraints apply in relation to time attributes and time expressions:</p>
+        <section id="timebase-section">
+          <h4><code>ttp:timeBase</code></h4>
+          <p>The only permitted <code>ttp:timeBase</code> value is <code>media</code>,
+            since <a href="#features-section"></a> prohibits all timeBase features
+            other than <a href="https://www.w3.org/TR/ttml2/#feature-timeBase-media"><code>#timeBase-media</code></a>.</p>
+          <p>This means that the beginning of the document timeline,
+            i.e. time "zero",
+             is the beginning of the <a>Related Media Object</a>.</p>
+        </section> <!-- timebase-section -->
+        <section id="timecontainer-section">
+          <h4><code>timeContainer</code></h4>
+          <p>The only permitted value of the <code>timeContainer</code> attribute is the default value, <code>par</code>.</p>
+          <p>Documents SHOULD omit the <code>timeContainer</code> attribute on all elements.</p>
+          <p>Documents MUST NOT set the <code>timeContainer</code> attribute to any value other than <code>par</code> on any element.</p>
+          <p class="note">This means that the <code>begin</code> attribute value for every timed element is relative to
+            the computed begin time of its parent element,
+            or for the <code>&lt;body&gt;</code> element, to time zero.</p>
+        </section> <!-- timecontainer-section -->
+        <section id="framerate-section">
+          <h4><code>ttp:frameRate</code></h4>
+          <p>If the document contains any time expression that uses the <code>f</code> metric,
+            or any time expression that contains a frames component,
+            the <code>ttp:frameRate</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element.</p>
+        </section> <!-- framerate-section -->
+        <section id="tickrate-section">
+          <h4><code>ttp:tickRate</code></h4>
+          <p>If the document contains any time expression that uses the <code>t</code> metric,
+            the <code>ttp:tickRate</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element.</p>
+        </section>
+        <section id="time-clock-with-frames-section" class="informative">
+          <h4><code>#time-clock-with-frames</code></h4>
+          <p>As specified in [[?ttml2]], a <code>#time-clock-with-frames</code> expression is translated to
+            a media time <code>M</code> according to
+            <br/>
+            <code>M = 3600 · hours + 60 · minutes + seconds + (frames ÷ (ttp:frameRateMultiplier · ttp:frameRate))</code>.</p>
+          <div class="example note">
+            <p>For example, the expression <code>01:23:45:15</code>,
```

Comment:
We should not encourage the use of the frame component. I vaguely remember that people used to confuse them with timecodes.

Reply:
I agree, but prohibiting it isn't worth the pain, and it's better to be clear about what it means. In particular, this note highlights (implicitly) the lack of dropMode.

Since you approved this pull request, I'm going to go ahead and merge - I assume if we want to make a recommendation about not using frames, we'll raise that as an additional issue.